examples/llama-bench/llama-bench.cpp

```diff
@@ -125,18 +125,15 @@ static std::string get_gpu_info() {
 #ifdef GGML_USE_SYCL
     int device_list[GGML_SYCL_MAX_DEVICES];
     ggml_sycl_get_gpu_list(device_list, GGML_SYCL_MAX_DEVICES);
```

Comment:
I think this can be removed now, `device_list` does not seem to be used anymore.

Reply:
yes, rm it.